docker/Dockerfile.build.aarch64-linux

```diff
@@ -76,6 +76,14 @@ RUN JPEG_TURBO_VERSION=2.0.5 && \
     make -j"$(grep ^processor /proc/cpuinfo | wc -l)" install 2>&1 >/dev/null && \
     rm -rf /libjpeg-turbo-${JPEG_TURBO_VERSION}
 
+# zstandard compression library
+RUN ZSTANDARD_VERSION=1.4.5 && \
+    curl -L https://github.com/facebook/zstd/releases/download/v${ZSTANDARD_VERSION}/zstd-${ZSTANDARD_VERSION}.tar.gz | tar -xzf - && \
+    cd zstd-${ZSTANDARD_VERSION} && \
+    CFLAGS="-fPIC" CXXFLAGS="-fPIC" CC=aarch64-linux-gnu-gcc CXX=aarch64-linux-gnu-g++ \
+      make -j"$(grep ^processor /proc/cpuinfo | wc -l)" install DESTDIR=/usr/aarch64-linux-gnu/ 2>&1 >/dev/null && \
```

Comment:
It would be better to use `prefix` instead of `DESTDIR` as  now it would install in
`/usr/aarch64-linux-gnu/usr/local/...`.
I have also noticed that tiff build cmd needs to be updated, CFLAGS and CXXFLAGS need to be set to `"-fPIC -I/usr/aarch64-unknown-nto-qnx/aarch64le/include"` to include a new dependency.

Reply:
You are right, DESTDIR and prefix have different behaviour. Regarding the flag for the tiff library, I am afraid you need to use the configure flag `--with-zstd-include-dir` and `--with-zstd-lib-dir` and not the `-I` because that, I tested on my host, doesn't work. Anyway I will make the modification later on

Reply:
I will make the modification only for the prefix. I think it should work then. Sorry for that, but I am working in the dark :-)

Reply:
Sure, no problem. We really appreciate your time and patience.
> Or perhaps, once changed from DESTDIR to prefix, the zstd will be found because now the prefix dir would be correct.

I think it won't work just like that as configure searches the default system directories, not the one we use in the prefix.

Reply:
It would be strange not checking the prefix dir set for the tiff library. Can we first have a try? If not working I can still add the `--with-zstd-*-dir` flags